### PR TITLE
Update velero to be compatible with 1.26

### PR DESCRIPTION
**Pull request by sbruzzese902**
Feats: 

- Updated velero to v1.11.0
- Updated plugins to v1.7.0
- Added compatibility with Kubernetes 1.26
- Added release notes for v.12 of dr module
- Changed required Terraform version to >= 1.3.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -332,9 +332,9 @@ kustomize build . | kubectl apply -f -
 [velero-aws-plugin-repo]: https://github.com/vmware-tanzu/velero-plugin-for-aws
 [velero-azure-plugin-repo]: https://github.com/vmware-tanzu/velero-plugin-for-microsoft-azure
 [velero-gcp-plugin-repo-permissions]: https://github.com/vmware-tanzu/velero-plugin-for-gcp#set-permissions-for-velero
-[kfd-velero-gcp-example]: https://github.com/sighupio/fury-kubernetes-dr/tree/master/example/gcp-example/main.tf
-[kfd-velero-aws-example]: https://github.com/sighupio/fury-kubernetes-dr/tree/master/example/aws-example/main.tf
-[kfd-velero-azure-example]: https://github.com/sighupio/fury-kubernetes-dr/tree/master/example/azure-example/main.tf
+[kfd-velero-gcp-example]: https://github.com/sighupio/fury-kubernetes-dr/tree/master/examples/gcp-examples/main.tf
+[kfd-velero-aws-example]: https://github.com/sighupio/fury-kubernetes-dr/tree/master/examples/aws-examples/main.tf
+[kfd-velero-azure-example]: https://github.com/sighupio/fury-kubernetes-dr/tree/master/examples/azure-examples/main.tf
```
Comment (nutellinoit) [inline README.md:337]:
```suggestion
[kfd-velero-gcp-example]: https://github.com/sighupio/fury-kubernetes-dr/tree/main/examples/gcp-examples/main.tf
[kfd-velero-aws-example]: https://github.com/sighupio/fury-kubernetes-dr/tree/main/examples/aws-examples/main.tf
[kfd-velero-azure-example]: https://github.com/sighupio/fury-kubernetes-dr/tree/main/examples/azure-examples/main.tf
```

File: README.md [hunk 2]
```diff
@@ -332,9 +332,9 @@ kustomize build . | kubectl apply -f -
...
 [kfd-velero-on-prem]: https://github.com/sighupio/fury-kubernetes-dr/tree/master/katalog/velero/velero-on-prem
```
Comment (nutellinoit) [inline README.md:338]:
```suggestion
[kfd-velero-on-prem]: https://github.com/sighupio/fury-kubernetes-dr/tree/main/katalog/velero/velero-on-prem
```

File: docs/releases/v1.12.0.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+kustomize build katalog/velero-aws | kubectl apply -f -
+# or
+kustomize build katalog/velero-gcp | kubectl apply -f -
+# or
+kustomize build katalog/velero-azure | kubectl apply -f -
+# and/or
+kustomize build katalog/velero-restic | kubectl apply -f -
+```
+
+For additional info you can have a look [here](https://velero.io/docs/v1.11/upgrade-to-1.11/)
```
Comment (nutellinoit) [inline docs/releases/v1.12.0.md:44]: Let's add something about the terraform requirement update

**Review by nutellinoit — CHANGES_REQUESTED** [review]
Why is this PR targeted from develop to main? (it's not a problem since this can be the PR to the release, but it should be always a branch -> develop)

Were the tests updated? Was this change tested with an -rc tag?

File: docs/releases/v1.12.0.md (hunk 3 above)
Comment (sbruzzese902) [reply docs/releases/v1.12.0.md]: Done!

**Comment by sbruzzese902** [comment]
I run the tests locally, but I can test with the -rc tag also.

**Review by nutellinoit — APPROVED** [review]
LGTM (let's wait before merging, since it is already the develop branch)

**Pull request closed without merging**